mllib/pom.xml

```diff
@@ -125,6 +125,10 @@
       <groupId>org.apache.spark</groupId>
       <artifactId>spark-test-tags_${scala.binary.version}</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.codehaus.janino</groupId>
+      <artifactId>janino</artifactId>
+    </dependency>
```

Comment:
I thought `sql` already has `org.codehaus` dependencies. Why do we need to add it here?